specification/azurelargeinstance/AzureLargeInstance.Management/models.tsp

```diff
@@ -408,6 +408,7 @@ model ForceState {
 model Tags {
   @projectedName("csharp", "tagsValue")
   @doc("Tags field of the AzureLargeInstance instance.")
+  #suppress "@azure-tools/typespec-azure-resource-manager/no-record" "Tags is an acceptable use."
   tags?: Record<string>;
 }
```

Comment:
@markcowl I feel like this should actually just reference `Foundations.ArmTagsProperty` in which case it wouldn't need the suppression.